homeassistant/components/light/tellstick.py

```diff
@@ -5,126 +5,84 @@
 https://home-assistant.io/components/light.tellstick/
 """
 from homeassistant.components.light import ATTR_BRIGHTNESS, Light
-from homeassistant.const import EVENT_HOMEASSISTANT_STOP
-
+from homeassistant.components.tellstick import (TellstickDevice,
+                                                TELLCORE_REGISTRY,
+                                                DISCOVER_DEVICES,
+                                                DISCOVER_CONFIG)
 REQUIREMENTS = ['tellcore-py==1.1.2']
 SIGNAL_REPETITIONS = 1
```

Comment:
Can you rename this to `DEFAULT_SIGNAL_REPITITIONS`. If this is shared among different tellstick components consider moving it to the main component.